code-style-intellij.xml

```diff
@@ -0,0 +1 @@
+<code_scheme name="Metacoder"/>
```

Comment:
What does this file do? I think we should maybe add this file also to the `.gitignore` (because we also ignore `.idea`)? Or we should also allow the codestyle file (`.idea/codeStyleSettings.xml`?), but this file alone is worthless.